mobile_app_connector/controllers/mobile_app_controller.py

```diff
@@ -106,6 +106,41 @@ def download_pdf(self, **parameters):
         host = request.env['ir.config_parameter'].get_param('web.base.url')
         letter_id = parameters['correspondenceid']
         letter = request.env['correspondence'].browse([int(letter_id)])
-        if letter.exists() and letter.letter_image:
```

Comment:
Please revert those changes, because I specifically added this check to avoid errors when there is no pdf image.